style/app.scss

```diff
@@ -644,7 +644,10 @@ a:active
 
 .opened
 {
-  width : 400px !important;
+  height           : 350px;
```

Comment:
Use `max-height` instead of `height` for this rule, to allow the component to shrink/grow depending on its contents.